ynput/ayon-applications
# Define WebActions

## Description
Applications addon does define web actions and adds new launch argument.

### Additional information
This PR requires
- AYON launcher 1.1.0 (https://github.com/ynput/ayon-launcher/pull/112)
- AYON core 0.4.4 (https://github.com/ynput/ayon-core/pull/781)
- AYON server 1.3.0 at least (https://github.com/ynput/ayon-backend/pull/183 & https://github.com/ynput/ayon-frontend/pull/563 )

This approach has few caveats. We always expected that launch of application happens from tray application where is possible to log into ftrack, kitsu etc. but with this approach they don't have option to login, we have to resolve that issue before full production.

### Testing notes
1. Make sure you have all requirements mentioned above.
2. Create package of applications addon.
3. Upload pacakge to server (upload is required).
4. Use it in a bundle.
5. If is used in developer bundle then make sure Develop mode is enabled and for production make sure Develop mode is disabled.
6. Select a task.
7. Applications actions should show up.
8. Launch any.

~~This PR might be split into 2 PRs, it does contain logic related to webactions and related to process preparation https://github.com/ynput/ayon-core/pull/781 .~~ They cannot be tested as separate PRs.
